packages/devtools_app/lib/src/memory/flutter/memory_heatmap.dart

```diff
@@ -240,19 +285,50 @@ class FlameChartRenderObject extends RenderBox {
     markNeedsPaint();
   }
 
+  List<String> _exposeMatchNames(String searchName) {
+    return matchNodeNames(_sizes.root.children, searchName);
+  }
+
+  /// Look for the node with a particular name (depth first traversal).
+  List<String> matchNodeNames(Map<String, Node> children, String searchName) {
+    final matches = <String>[];
+    final matchName = searchName.toLowerCase();
+    for (var child in children.entries) {
+      final node = child.value;
+      if (node.children.isEmpty) {
+        final lcNodeName = node.name.toLowerCase();
+        if (!lcNodeName.endsWith('.dart') && lcNodeName.startsWith(matchName)) {
```

Comment:
replace startsWith with contains? Here and below

Reply:
I've changed it to contains but I still prefer startswith especially since I know the classNames auto-complete shows more pertinent information earlier.  May want a setting for auto-complete that defaults to either start or contains.